client/app/routes/collection/add.js

```diff
@@ -1,4 +1,124 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+
+    model() {
+        return {
+            submission_form_name: 'Preprints Submission Form',
+            sections: [
+                'upload',
+                'disciplines',
+                'basic info',
+                'authors',
+                'submit'
+            ],
+            initial_state: {
+                upload_section: {
+                    state: ['unsaved', 'editing'],
```

Comment:
Anything could have multiple states. Most would only have one, but some have multiple things to keep track of about that parameter. The upload section needs to keep track of whether or not it's saved, and whether or not its being edited.